### PR TITLE
Add cloud related release notes

### PR DESCRIPTION
**Pull request by xlight05**
## Purpose
$Subject.

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

**Review by anuruddhal — APPROVED** [review]

File: release-notes/swan-lake-2201.1.0-release-note.md [hunk 1]
```diff
@@ -136,16 +136,26 @@ To view bug fixes, see the [GitHub milestone for Swan Lake 2201.1.0](https://git
...
 #### Bug Fixes
 
 To view bug fixes, see the GitHub milestone for Swan Lake 2201.1.0 of the repositories below.
 
 - [C2C](https://github.com/ballerina-platform/module-ballerina-c2c/issues?q=is%3Aissue+is%3Aclosed+label%3AType%2FBug+milestone%3A%22Ballerina+Swan+Lake+-+2201.1.0%22)
 
+### AWS Lambda
+
+#### Improvements
+- Removed from the ballerina distribution. For existing projects with Dependencies.toml change the version into 2.1.0.
```
Comment (praneesha) [inline release-notes/swan-lake-2201.1.0-release-note.md:152]:
```suggestion
- Removed from the Ballerina distribution. For existing projects, change the version to 2.1.0 in the `Dependencies.toml` file.
```
Removed what? Please specify.

File: release-notes/swan-lake-2201.1.0-release-note.md [hunk 2]
```diff
@@ -136,16 +136,26 @@ To view bug fixes, see the [GitHub milestone for Swan Lake 2201.1.0](https://git
...
+
+### Azure Functions
+
+#### Improvements
+- Removed from the ballerina distribution. For existing projects with Dependencies.toml change the version into 2.1.0.
```
Comment (praneesha) [inline release-notes/swan-lake-2201.1.0-release-note.md:157]:
```suggestion
- Removed from the Ballerina distribution. For existing projects, change the version to 2.1.0 in the `Dependencies.toml` file .
```
Removed what? Please specify.

**Review by praneesha — APPROVED** [review]

File: release-notes/swan-lake-2201.1.0-release-note.md [hunk 3]
```diff
@@ -136,16 +136,26 @@ To view bug fixes, see the [GitHub milestone for Swan Lake 2201.1.0](https://git
...
+- Reduced the package size
+- Docker image generation now relies on the user's docker client
 
 #### Bug Fixes
 
 To view bug fixes, see the GitHub milestone for Swan Lake 2201.1.0 of the repositories below.
 
 - [C2C](https://github.com/ballerina-platform/module-ballerina-c2c/issues?q=is%3Aissue+is%3Aclosed+label%3AType%2FBug+milestone%3A%22Ballerina+Swan+Lake+-+2201.1.0%22)
 
+### `ballerinax/awslambda` Package
```
Comment (praneesha) [inline release-notes/swan-lake-2201.1.0-release-note.md:149]:
```suggestion
### `ballerinax/awslambda` package
```

File: release-notes/swan-lake-2201.1.0-release-note.md [hunk 4]
```diff
@@ -136,16 +136,26 @@ To view bug fixes, see the [GitHub milestone for Swan Lake 2201.1.0](https://git
...
+
+#### Improvements
+- Removed the package from the Ballerina distribution. For existing projects, change the version to `2.1.0` in the `Dependencies.toml` file.
+
+### `ballerinax/azure_functions` Package
```
Comment (praneesha) [inline release-notes/swan-lake-2201.1.0-release-note.md:154]:
```suggestion
### `ballerinax/azure_functions` package
```

**Review by praneesha — APPROVED** [review]

**Pull request merged**